src/Bundle/Resources/config/services/state.xml

```diff
@@ -13,6 +13,24 @@
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
+        <service id="sylius.resource_factory.expression_language" class="Symfony\Component\ExpressionLanguage\ExpressionLanguage" />
```

Comment:
I fail to see connection of these changes with plural naming

Reply:
Hum indeed, I'll check that.